genrl/deep/common/trainer.py

```diff
@@ -369,7 +369,7 @@ def train(self) -> None:
             # update params for DQN
             if self.agent.__class__.__name__ == "DQN":
                 if self.agent.replay_buffer.pos > self.agent.batch_size:
-                    self.agent.update_params()
+                    self.agent.update_params(timestep)
 
                 if timestep % self.update_interval == 0:
                     self.agent.update_target_model()
```

Comment:
Do you think there's a way to combine the DQN if statement with the others? Thats the only last dependency, I guess

Reply:
Yea I am thinking of a workaround for this.

Reply:
We can do one thing we can pass pos and batchsize to update params, they won't be used for other agents, idk this might be bad coding style because already timestep is not used for some agents.